### DISCUSSION TITLE
1.1.0에서 질문있습니다.

### DISCUSSION THREAD

**Discussion by swoo9799** [changdoc/naver-vaccine-macro#168] [Q&A]
원래방식이 병원 하나 지정해서 돌리는걸로 잘 이용하고 있습니다.
이번에 네이버 지도에서 뜨는방식이랑 카카오에서 뜨는 방식이 업데이트 된걸 보았는데
네이버 지도에서 뜨는것과 병원 지정해서 뜨는걸 같이 돌리면 밴을 먹는지 궁금합니다.
또한 지도에서 뜨는걸 캐치하는방식과 지정해서 뜨는 방식의 속도차이도 있어서 두개 다 써야 확률이 높은지도 궁금합니다.!
이번 업데이트 정말 감사합니다!

**Comment by Dddddddssd** [comment]
개발자님은 아니지만 저도 비슷한 상황을 겪어봐서 답변드립니다.
저같은 경우에는 어제 지도랑 병원 지정 동시에 돌리니까 락 걸리더군요. 하나만 새로고침 하시는걸 추천드립니다.

**Comment by swoo9799** [comment]
아 또 질문이 1.1.0의 네이버랑 카카오톡 잔여백신 매크로를 같이 사용해도 되나요?

**Reply by swoo9799** [reply]
돌려보니깐 네이버 지도에서 새로고침이 안되네요
